ui/src/components/AppDetailsToolbar.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { Toolbar, ToolbarGroup, ToolbarItem, Breadcrumb, BreadcrumbItem, Button } from '@patternfly/react-core';
+import './AppDetailedToolbar.css';
+
+export class AppDetailsToolbar extends React.Component {
+  onDisableApp = () => {
+    console.log('Disable button clicked');
+  }
+
+  onHomeClick = () => {
+    this.props.history.push('/');
+  }
+
+  onSaveApp = () => {
+    console.log('Save button clicked');
+  }
+
+  render () {
+    return (
+      <Toolbar className='details-toolbar'>
+        <ToolbarGroup>
+          <ToolbarItem>
+            <Breadcrumb>
+              <BreadcrumbItem className='home-breadcrumb' onClick={this.onHomeClick}>Home</BreadcrumbItem>
+              <BreadcrumbItem isActive >{this.props.app.appName}</BreadcrumbItem>
+            </Breadcrumb>
+          </ToolbarItem>
+        </ToolbarGroup>
+        <ToolbarGroup className='toolbar-buttons'>
+          <Button className='toolbar-button' onClick={this.onDisableApp} variant="primary">Disable App</Button>
+          <Button className='toolbar-button' onClick={this.onSaveApp} variant="primary">Save</Button>
```

Comment:
Re spacing on between the buttons, I see patternfly uses the following to add spacing between them, fyi
```
<Button variant="secondary">Secondary</Button>{' '}
<Button variant="tertiary">Tertiary</Button>
```

Reply:
Hmm. I've tested that and it doesn't seem to work. Is it necessary to switch from the current implementation?